docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx

```diff
@@ -18,12 +18,6 @@ keywords:
   - typescript
 ---
 
-:::info
```

Comment:
Please revert this. See [internal discussion](https://o1-labs.slack.com/archives/C0206B5S80Y/p1714739315360389?thread_ts=1714734545.985919&cid=C0206B5S80Y).